src/Command/AnalyzeCommand.php

```diff
@@ -84,7 +85,8 @@ protected function execute(
         new ConsoleFormatter($this->dispatcher, $output);
 
         $parser = new NikicPhpParser();
-        $astMap = $this->astRunner->createAstMapByFiles($parser, $this->dispatcher, $this->collectFiles($configuration));
+        $files = $this->collectFiles($configuration);
+        $astMap = $this->astRunner->createAstMapByFiles($parser, $this->dispatcher, $files);
```

Comment:
no need to declare this variable?

Reply:
It's more readable and `collectFiles` should move to its own class. There will be a following PR